### DISCUSSION TITLE
Using Math in Github Pages

### DISCUSSION THREAD

**Discussion by N3XT14** [community/community#45178] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi All,

I will get to the point. I want to insert Math functions like log, sqrt, sum, etc. in my github pages but I am unable to. I know that Github supports the  Latex bydefault but still it is not working

My markdown file shows it properly but when it is hosted as github pages it doesn't read the library or it fails to recognize

This is the link
https://n3xt14.github.io/DSA/DSAMathDocs

and my repository code is 
https://raw.githubusercontent.com/N3XT14/DSA/main/DSAMathDocs.md

Please let me know what is the issue and how can I fix it

**Comment by jge162** [comment]
@N3XT14 maybe, I am not 100% SURE its possible that the Jekyll theme you are using for your GitHub Pages site does not have built-in support for LaTeX?? In this case, you can try using a different theme that does support LaTeX, or you can try using a plugin to add LaTeX support to your current theme. (you need to verify sorry I cant test)...

Another issue could be that your markdown file is not properly formatted? I can see that you are using standard markdown syntax to write the math expressions, but you need to use the LaTeX syntax. For example, sqrt(2) should be written as $\sqrt{2}$ in your markdown file.

...

**Reply by N3XT14** [reply]
Thanks for the reply. But that is what I am looking for instead of writing symbols which is quite harder in case of Mathematical summation form or logarithms or Limits.

Thats the reason I am looking for a way which does this conversion in backend. 
Regarding the example, those I have picked from the official github site but it is not working in my case so I want to find out what am I missing here
